src/Java.Interop.Tools.Generator/Metadata/FixupXmlDocument.cs

```diff
@@ -168,6 +169,18 @@ public void Apply (ApiXmlDocument apiDocument, string apiLevelString, int produc
 			}
 		}
 
+		public List<NamespaceTransform> GetNamespaceTransforms ()
+		{
+			var list = new List<NamespaceTransform> ();
+
+			foreach (var xe in FixupDocument.XPathSelectElements ("/metadata/ns-replace")) {
+				if (NamespaceTransform.TryParse (xe, out var transform))
+					list.Add (transform);
```

Comment:
What should happen if there's a duplicate?  Should we care? Emit a warning?

*Particularly* if we're thinking of adding `Metadata.xml` into NuGet packages for "reuse" by "downstream binding projects", the likelihood for duplicates will increase.  It shouldn't be an *error*, but would a warning be useful?  Or would it be too common for warnings?

Reply:
I think it's fine to ignore duplicates, we do the same for other `metadata`.

I'm not aware of any plan to add `Metadata.xml` into NuGet packages.